draft-ietf-quic-multipath.md

```diff
@@ -319,7 +319,11 @@ Path ID 0.
 After the handshake concluded if negotiation of multipath support succeeded,
 endpoints SHOULD use PATH_ACK frames instead of ACK frames,
 also for acknowledging so far unacknowledged 0-RTT packets, using
-Path ID 0. Similarly after a successful handshake, endpoints SHOULD also use
+Path ID 0. Further more, endpoints SHOULD use PATH_ACK frames in all 1-RTT packets after
+the successful negotiation for multipath. If endpoints receive ACK frame
+after a succesful negotiation resulting from packet out-of-order arrival,
```

Comment:
Given this is a SHOULD only, it is not only out-of-order arrivals. The peer can decide to keep using ACK frames for all Path ID 0 packets. However, we should explain when this might be useful, e.g. in order to same some bytes, especially when only one path is used.